dropwizard-auth/src/main/java/io/dropwizard/auth/AuthFilter.java

```diff
@@ -138,6 +138,7 @@ protected boolean authenticate(ContainerRequestContext requestContext, C credent
                 return false;
             }
 
+            boolean secure = requestContext.getSecurityContext().isSecure();
```

Comment:
> can you make this `final`?

Sure.

> Also, do we need to check that a SecurityContext has been set before calling this to avoid a NPE?

Good question. I searched a bit and found that such a null check is done in the Jersey examples of how to implement a [ContainerRequestFilter](https://jersey.java.net/documentation/latest/filters-and-interceptors.html#d0e9666). So yes, I think we should do it too.